libs/vkd3d/command.c

```diff
@@ -18168,6 +18168,9 @@ static bool d3d12_command_queue_needs_staggered_submissions_locked(struct d3d12_
     if (command_queue->vkd3d_queue->command_queue_count == 1)
         return false;
 
+    if (vkd3d_config_flags & VKD3D_CONFIG_FLAG_NO_STAGGERED_SUBMIT)
```

Comment:
I think we'll want to flip this into opt-in until the regression in Wukong is solved?

Reply:
done.